### ISSUE TITLE
About page

### ISSUE THREAD

**Issue by nudgeD** [a8cteam51/kelsey-montague#6]
Current page: https://kelseymontagueart.com/about/    

- Team members
   - Featuring Kelsey at the top (image and bio)
   - Courtney Montague below (image and bio) in the same format.
- Section below the team member profiles 
    - list out people and places that Kelsey has worked with. Update the design/layout of this to be more attractive and engaging than it is now.

**Comment by rpasillas** [comment]
Team members will be basic Gutenberg content blocks instead of a "mini" archive of a custom post type yeah?

**Comment by nudgeD** [comment]
Yes, that was my thinking. Maybe with a class for each member and/or a custom template.
